.github/workflows/Test.yml

```diff
@@ -85,7 +85,7 @@ jobs:
           sudo apt update -qq
           sudo apt install -y gnat
           python -m pip install --upgrade pip
-          python -m pip install black wheel
+          python -m pip install black wheel pyTooling
```

Comment:
Was this still needed or is it now handled via pyproject.toml?

Reply:
@umarcor I think for now ok, as we still build with `python setup.py ...` instead of using latest Python build system.